app/controllers/crimes_controller.rb

```diff
@@ -33,6 +33,15 @@ def show
     @user = User.find(@crime.user.id)
   end
 
+  def my_crimes
+    @my_crimes = []
+    Crime.all.map do |crime|
```

Comment:
I think this should be
`@my_crimes = Crime.all.select { |crime| crime.user == current_user }`